package ru.llm.pivocore.exception;

public class ReviewException extends RuntimeException {

    public ReviewException(String message, Throwable cause) {
        super(message, cause);
    }

}
